src/api/taxonomyApi.ts

```diff
@@ -19,6 +20,28 @@ interface FetchTopicResourcesParams {
   filters?: string;
   subjectId?: string;
 }
+interface TaxonomyTranslation {
+  name: string;
+  language: string;
+}
+
+interface TaxonomyMetadata {
+  customFields: Record<string, string>;
+  grepCodes: string[];
+  visible: boolean;
+}
+
+export interface Subject {
+  contentUri: string | null;
+  id: string;
+  metadata: TaxonomyMetadata;
+  name: string;
+  path: string | null;
```

Comment:
Path kan være tom streng, men vil alltid være satt.

Reply:
Hmm, er det nytt? 
Hvorfor vil vi ha tom streng istedet for `null`?

Reply:
@Jonas-C ville ha det for å ha bedre typing av taksonomi. Er med i siste taksonomi.

Reply:
Personlig så syns jeg det er dårligere typing å tvinge noe til å være tom streng når det er en verdi som ikke finnes, men skal lære meg å leve med det.

Reply:
Tanken bak det var at man fikk en mer konsistent håndtering av path. I frontenden nå tilsvarer bruk av `path` nesten alltid `.path ?? ''`. I React Router 6 er udefinerte `to`-lenker en throwable feil, så inntil frontend-packages er bedre typet slipper vi enklere unna feil med denne fremgangsmåten. Enig i at det ikke nødvendigvis er optimalt.